frontend/src/components/SignForms/SignInForm/FormikInContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Remove please this file and move all the logic inside SignIn container
Rename please all the path to the: `frontend/src/pages/SignIn/SignInContainer.tsx` and `frontend/src/pages/SignIn/SignIn.tsx`

Reply:
_**Remove please this file and move all the logic inside SignIn container**_ 
Lets see, in future I will use useQuery and i want to add this into FormikContainer - now SignIn already include over 70n lines, and its almost all only Sign-form describe. If i will add logic yet it will be not fine, this file wll include over then 120-150 lines. Maybe i will rename FormikContainer to SignContainer and

**_Rename please all the path to the: frontend/src/pages/SignIn/SignInContainer.tsx and frontend/src/pages/SignIn/SignIn.tsx_**
will rename it as SignPage.tsx and also will fix path.

Reply:
You should have SignIn folder with SignIn.tsx and SignInContainer.tsx inside it. Now we have too much extra folders and components here

Reply:
Ok, I see that folders-hell))) replaced and fixed paths and remove some files

Reply:
SignInForm is not a reusable component, so it rather should be in pages/SignIn folder